host-bin/mount-chroot

```diff
@@ -145,10 +150,28 @@ for NAME in "$@"; do
             fi
         fi
 
-        # Detect the key file
+        # Detect unencrypted keys
+        if [ -z "$UKEYFILE" ]; then
+            UKEYFILE="/home/chronos/user/crouton/$NAME.chkey"
```

Comment:
Why make ```~/crouton``` visible? If it is for storing keys, I think ```~/.crouton``` will be more suitale. Just like ```~/.ssh```